example/burrow_integration/get*Transactions.py

```diff
@@ -0,0 +1,269 @@
+import os
+import binascii
+from iroha import IrohaCrypto
+from iroha import Iroha, IrohaGrpc
+from iroha.primitive_pb2 import can_set_my_account_detail
+import sys
+from Crypto.Hash import keccak
+
+if sys.version_info[0] < 3:
+    raise Exception("Python 3 or a more recent version is required.")
+
+# Here is the information about the environment and admin account information:
+IROHA_HOST_ADDR = os.getenv("IROHA_HOST_ADDR", "127.0.0.1")
+IROHA_PORT = os.getenv("IROHA_PORT", "50051")
+ADMIN_ACCOUNT_ID = os.getenv("ADMIN_ACCOUNT_ID", "admin@test")
+ADMIN_PRIVATE_KEY = os.getenv(
+    "ADMIN_PRIVATE_KEY",
+    "f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70",
+)
+
+iroha = Iroha(ADMIN_ACCOUNT_ID)
+net = IrohaGrpc("{}:{}".format(IROHA_HOST_ADDR, IROHA_PORT))
+
+test_private_key = IrohaCrypto.private_key()
+test_public_key = IrohaCrypto.derive_public_key(test_private_key).decode("utf-8")
+
+
+def trace(func):
```

Comment:
Many functions which You are using are here:
https://github.com/hyperledger/iroha/blob/main/example/burrow_integration/integration_helpers.py
I suggest not to copy-paste them, but use existing implementations